syntax = "proto3";

package data_relay_grpc.grpc.proto;

service PingService {
  rpc Ping (PingRequest) returns (PingResponse);
}

message PingRequest {}
message PingResponse {}
